spec/controllers/wikis_controller_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe WikisController do
+  include Devise::TestHelpers
+
+  before do
+    @user = build(:user)
+    #to be created!
+    # @wiki = build(:wiki)
+    visit root_path
+     click_link ('Sign In')
```

Comment:
capybara helpers are for feature specs, driving the interface as a virtual user. controller specs are unit tests, testing only the *code*, the unit. In controller specs you usually send GET/POST/PATCH/DELETE requests and test the response